bin/tools/Templates/FrontendUnitTest.js

```diff
@@ -1278,6 +1315,14 @@ export class FrontendUnitTest extends AbstractTemplate {
    * @returns {String}
    * @constructor
    */
+  static get DIRECTIVE_TPL_PATH() {
+    return path.join(__dirname, '../frontend-tests/tpl/directive.twig');
+  }
+
+  /**
+   * @returns {String}
+   * @constructor
+   */
   static get DIRECTIVE_AS_CLASS_TPL_PATH() {
     return path.join(__dirname, '../frontend-tests/tpl/directive_as_class.twig');
```

Comment:
Please keep the structure of generic microservice. In this case, I would put this template in [root_folder]/tpl/tests/frontend/directive_as_class.twig ;)